axiomtradeapi/client.py

```
@@ -579,7 +591,7 @@ def buy_token(self, private_key: str, token_mint: str, amount: float,
             response = requests.post(
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)